dashboard/app/models/level.rb

```diff
@@ -59,8 +62,13 @@ def self.model_name
 
   # https://github.com/rails/rails/issues/3508#issuecomment-29858772
   # Include type in serialization.
+  # also include level_concept_difficulty
```

Comment:
Reading the link, it isn't clear to me why this is necessary for level_concept_difficulty.